mozilla-release/mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/ActivityStreamPanel.java

```diff
@@ -140,10 +147,23 @@ public ActivityStreamPanel(final Context context, AttributeSet attrs) {
         customizeNewTabViewSnackBar = findViewById(R.id.customize_newtab_snackbar);
 
         final View customizeTabLink = customizeNewTabView.findViewById(R.id.customize_tab_link);
+        if (abManager.getHomeSettingsStyle() == "text") {
+            customizeTabLink.setBackgroundColor(Color.argb(0, 0, 0, 0));
+        }
         customizeTabLink.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 final Intent intent = new Intent(context, GeckoPreferences.class);
+                final GeckoBundle signal = new GeckoBundle();
```

Comment:
I need to add similar signal on load. Where should I add it?

Reply:
Why?

Reply:
In the ticket https://cliqztix.atlassian.net/browse/AB2-667 there are 2 signals I need to add, one on click and one on show. Is the constructor the right place to send the telemetry for show action?

Reply:
@ravjit-cliqz you implemented the auto disappearing thing, can you help here?